### PR TITLE
add depart with distance

### LINKED ISSUE #172: Add a "Head {direction} on {way_name} for {distance}" phrase
When giving a direction like `Continue on 16th street` or `Head south on St James Boulevard` it would be nice to optionally add on the distance. We can currently do this for the [continue](https://github.com/Project-OSRM/osrm-text-instructions/blob/7baf2ec6c9770f65bdd8b39f21cbf4caa88b1553/languages/translations/en.json#L97) but it would be nice to be able do this departures as well.

### PR DESCRIPTION
**Pull request by lyzidiamond**
Closes #172.

## Tasklist

 - [x] Add translation entry in `en.js`
 - [x] Add tests for continue with distance and depart with distance
 - [x] Add changelog entry
 - [x] Review

cc @mcwhittemore

### CODE REVIEW

File: languages/translations/en.json [hunk 1]
```diff
@@ -125,7 +125,8 @@
         "depart": {
             "default": {
                 "default": "Head {direction}",
-                "name": "Head {direction} on {way_name}"
+                "name": "Head {direction} on {way_name}",
+                "namedistance": "Head {direction} on {way_name} for {distance}"
```
Comment (1ec5) [inline languages/translations/en.json]: In the absence of #155, do we need to also add this line to all the other translation files?

Comment (mcwhittemore) [reply languages/translations/en.json]: hmmm... if we're going to do that, I'd like to use `depart name` rather than the english here.

Comment (lyzidiamond) [reply languages/translations/en.json]: I did this!

**Comment by lyzidiamond** [comment]
Okay @mcwhittemore one last review and then we should be able to merge here.

**Pull request merged**